### PR TITLE
added everything needed for camera

### PR DESCRIPTION
**Pull request by smileyenot983**
Everything works as it should work, but you should first make it compile in python 3+

### CODE REVIEW

File: sensor_camera/.idea/sensor_camera.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (l1va) [inline sensor_camera/.idea/sensor_camera.iml]: add .idea to gitignore

File: sensor_camera/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+## Generate actions in the 'action' folder
+# add_action_files(
+#   FILES
+#   Action1.action
+#   Action2.action
+# )
+
+## Generate added messages and services with any dependencies listed here
+# generate_messages(
+#   DEPENDENCIES
```
Comment (l1va) [inline sensor_camera/CMakeLists.txt]: do you need all these comments?

File: sensor_camera/scripts/subscriber.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	rospy.spin()
+counter = 0
+
+def increment():
+	global counter
+	counter=counter+1
+
+def image_callback(msg):
+
+	path = '/home/ramil/robot_ws/src/sensor_camera/images/pic' + str(counter) + '.png'
```
Comment (l1va) [inline sensor_camera/scripts/subscriber.py]: hardcode that will not work on my laptop

File: sensor_camera/CMakeLists.txt (hunk 2 above)
Comment (smileyenot983) [reply sensor_camera/CMakeLists.txt]: yes, of course

File: sensor_camera/scripts/subscriber.py (hunk 3 above)
Comment (smileyenot983) [reply sensor_camera/scripts/subscriber.py]: fixed

File: sensor_camera/.idea/sensor_camera.iml (hunk 1 above)
Comment (smileyenot983) [reply sensor_camera/.idea/sensor_camera.iml]: added

**Pull request merged**